h/models/document/_document.py

```diff
@@ -152,7 +152,7 @@ def merge_documents(session, documents, updated=None):
             m.document = master
             m.updated = updated
 
-    try:
+    try:  # pylint:disable=too-many-try-statements
```

Comment:
I think this one should be split up. I think the only statement that could cause an integrity error is the flush. The other commands are queries.

Reply:
Not sure, the first one is an update of `document_id` which could raise? The deletion probably won't but between a try block with two statements + plus a separate deletion or everything in the same block... I think I'd take thing as it is now.